Tools/CISupport/ews-build/steps.py

```diff
@@ -1456,27 +1456,34 @@ def hideStepIf(self, results, step):
 
 
 class Trigger(trigger.Trigger):
-    def __init__(self, schedulerNames, include_revision=True, triggers=None, **kwargs):
+    def __init__(self, schedulerNames, include_revision=True, triggers=None, patch=False, pull_request=False, **kwargs):
```

Comment:
It seems weird to have optional parameters (patch and pull_request) with default values, and have almost invalid configuration in those default values. It would be an error to have both patch and pull_request as False.

Let's either make these parameters mandatory, or make either patch or pull_request to be true by default (preferably patch for now).